codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithypython/awssdk/shapevisitor/conversionwriters/AwsSdkToDafnyConversionFunctionWriter.java

```diff
@@ -467,18 +477,18 @@ public void writeStringEnumShapeConverter(
                 // with `elif`
                 shouldOpenNewIfBlock ? "if" : "elif",
                 dataSourceInsideConversionFunction,
-                value,
+                name,
```

Comment:
```suggestion
                value,
```

Try this out, ditto for the other conversion.

Right now, this is generated:

```
def com_amazonaws_s3_OptionalObjectAttributes(native_input):
    # Convert OptionalObjectAttributes
    if native_input == "RESTORE_STATUS":
        return OptionalObjectAttributes_RESTORE__STATUS()
    else:
        raise ValueError("No recognized enum value in enum type: " + native_input)
```

But I believe it should be 

```
def com_amazonaws_s3_OptionalObjectAttributes(native_input):
    # Convert OptionalObjectAttributes
    if native_input == "RestoreStatus":
        return OptionalObjectAttributes_RESTORE__STATUS()
    else:
        raise ValueError("No recognized enum value in enum type: " + native_input)
```

Because the native value according to boto3 is `"RestoreStatus"`: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/list_objects_v2.html